website/_config.yml

```diff
@@ -0,0 +1,40 @@
+# General configuration.
+title: Error Prone Support
+logo: assets/images/favicon.svg
+url: https://error-prone.picnic.tech
+description: >-
+  Error Prone extensions: extra bug checkers and a large battery of Refaster templates.
+
+remote_theme: just-the-docs/just-the-docs
```

Comment:
Okay, the build now [fails](https://github.com/PicnicSupermarket/error-prone-support/actions/runs/3169220790/jobs/5160972905) with `Could not find 'just-the-docs' (>= 0) among 148 total gem(s) (Gem::MissingSpecError)`. I guess there's a difference between my local and the GHA setup. To be investigated.

Reply:
IIUC it happens because [this image](https://github.com/actions/jekyll-build-pages/blob/9c76bbb6c7b6fdbfde12014ed64206062b8eeac6/action.yml#L29-L31) doesn't run `bundle install`. That would also mean that the `Gemfile` contents are misleading. [This comment](https://github.com/actions/jekyll-build-pages/issues/46#issuecomment-1215505246) suggests an alternative approach. Not sure I'll get around to trying this today still, though...

Reply:
Most actions run in the scope of a Docker image IIUC, so that means indeed `bundle install` of our own `gemfile` isn't run 😞